src/cli/rdb-cli.c

```diff
@@ -354,9 +382,22 @@ int readCommonOptions(RdbParser *p, int argc, char* argv[], Options *options, in
             continue;
         }
 
+        if (getOptArg(argc, argv, &at, "-e", "--expired", NULL, NULL)) {
+            if ((applyFilters) && (!RDBX_createHandlersFilterExpired(p, 0)))
+                exit(RDBX_ERR_FAILED_CREATE_FILTER);
+            continue;
+        }
+
+        if (getOptArg(argc, argv, &at, "-E", "--no-expired", NULL, NULL)) {
+            if ((applyFilters) && (!RDBX_createHandlersFilterExpired(p, 1)))
+                exit(RDBX_ERR_FAILED_CREATE_FILTER);
```

Comment:
Plase be aware exit code is a single byte. If you pass a larger integer, it will be sliced. If I'm not mistaken, this is an integer larger than 255.